### ISSUE TITLE
Fix: Svelte parser disallows valid IDL attribute names such as `defaultValue`

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#2590]
- Markuplint version: `4.x.x`
- Parser lang: Svelte
- Node.js version: `22.x.x`
- OS: macOS

## Describe the bug

In Svelte templates, it is common to use attribute names that correspond to IDL attributes, such as `defaultValue` or `indeterminate`, especially when using directives or bindings. However, the current behavior of the Markuplint Svelte parser disallows these attributes, incorrectly treating them as invalid because they match IDL attribute names.

This limitation results in false positives when using valid attributes that are intentionally written in the markup.

## Code Example or Playground URL

```html
<input type="text" defaultValue="foo" />
```

## Steps To Reproduce

1. Use attribute names like `defaultValue` in a Svelte template.
2. Run markuplint with the Svelte parser.
3. Observe a lint error indicating that these attributes are not allowed.

## The current behavior

Attribute names matching certain IDL attributes are reported as invalid in Svelte templates, even though their usage is valid.

## The expected behavior

Markuplint should allow attribute names that correspond to valid IDL attributes when used explicitly in Svelte templates, as their usage is valid and intentional.
